lib/bundlex/toolchain/common/compilers.ex

```diff
@@ -1,30 +1,10 @@
 defmodule Bundlex.Toolchain.Common.Compilers do
-  defstruct [:c, :cpp]
+  @moduledoc """
```

Comment:
As this is not a part of the public API, moduledoc should be set to false, and needed description should be placed right below. Docs for functions can be added normally.